x-pack/legacy/plugins/alerting_ui/np_ready/public/application/lib/doc_title.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+
+class DocTitleService {
+  private changeDocTitle: any = () => {};
+
+  public init(changeDocTitle: any): void {
+    this.changeDocTitle = changeDocTitle;
+  }
+
+  public setTitle(page?: string): void {
+    let updatedTitle = i18n.translate('xpack.alertingUI.home.breadcrumbTitle', {
```

Comment:
maybe move this to the `default` case in the switch?

Reply:
Fixed in https://github.com/elastic/kibana/pull/47843/commits/85578a4fc084818300dbc9fe472dab2050dce9a5.